manifests/params.pp

```diff
@@ -26,9 +26,15 @@
         $rake_package     = 'rubygem-rake'
         $rake_provider    = 'yum'
       }
-      $bundler_ensure   = 'installed'
-      $bundler_package  = 'bundler'
-      $bundler_provider = 'gem'
+      if versioncmp($::operatingsystemmajrelease, '7') < 0 {
```

Comment:
This is confusing with the if statement above for EL5. Suggest refactoring into a case statement on `operatingsystemmajrelease` with a default case that fails for unsupported systems.